.github/workflows/unit-tests.yaml

```
@@ -11,25 +11,37 @@ permissions:
-              - '!README.md'
-              - '!docs/**'
+          patterns: |
+            !.spelling
+            !README.md
+            !docs/**
+            !.spectral.yaml
+            !api/**/openapi.yaml
+
+  unit-tests:
+    needs: check-changes
+    if: ${{ needs.check-files.outputs.src_changed == 'true' }}
```

actionlint (1.7.4)

property "check-files" is not defined in object type {check-changes: {outputs: {src_changed: string}; result: string}}

(expression)